src/js/components/AppHealthBarWithTooltipComponent.jsx

```diff
@@ -0,0 +1,79 @@
+var React = require("react/addons");
+
+var TooltipMixin = require("../mixins/TooltipMixin");
+var AppHealthBarComponent = require("./AppHealthBarComponent");
+var AppHealthBreakdownComponent = require("./AppHealthBreakdownComponent");
+var Util = require("../helpers/Util");
+var HealthStatus = require("../constants/HealthStatus");
+
+var appHealthBreakdownFields = [
+  {label: "Healthy", key: "healthy", state: HealthStatus.HEALTHY},
+  {label: "Unhealthy", key: "unhealthy", state: HealthStatus.UNHEALTHY},
+  {label: "Unknown", key: "unknown", state: HealthStatus.UNKNOWN}
+];
```

Comment:
Yeah, this is a tricky one. It's not really a constant so much as a configuration which happens to be the same. You're definitely right that it shouldn't be repeated - I might adopt the same map system used in the health bar and just pass the constants.